bin/copyRawDataToPrm.sh

```diff
@@ -260,7 +249,7 @@ function splitSamplesheetPerProject() {
 	local _rsyncControlFileFinished="${PRM_ROOT_DIR}/logs/${_run}/run01.${SCRIPT_NAME}.finished"
 	local _controlFileBase="${PRM_ROOT_DIR}/logs/${_run}/run01.splitSamplesheetPerProject"
 	local _logFile="${_controlFileBase}.log"
-	
+	#
```

Comment:
Why would you have only a hashtag when it is not  a comment?

Reply:
Silly workaround for loosing indentation of empty lines by others using other editors. I've removed all indented blank lines and replaced them with comments. Most of them with some additional text, a few without; those are just to separate two blocks of code.